source/content/guides/multisite/07-search-replace.md

```diff
@@ -0,0 +1,50 @@
+---
+title: WordPress Multisite
+subtitle: Search-Replace
+description: Additional handing required for running search-replace on WordPress Multisite.
+type: guide
+getfeedbackform: default
+contenttype: [guide]
+innav: [false]
+categories: [cms, troubleshooting]
+cms: [wordpress]
+audience: [development]
+product: [--]
+integration: [--]
+tags: [multisite]
+permalink: docs/guides/multisite/search-replace/
+editpath: multisite/07-search-replace.md
+---
+
+<Alert title="Early Access" type="info" icon="leaf">
+
+WordPress Multisite (WPMS) Search-Replace is available for [Early Access](/oss-support-levels#early-access) participants. Features for WPMS Search-Replace are in active development. Pantheon's development team is rolling out new functionality often while this product is in Early Access. Visit [#wordpress in our community Slack](https://pantheon-community.slack.com/archives/CT8MC5Y0K) (you can sign up for the [Pantheon Slack channel here](https://slackin.pantheon.io/) if you don't already have an account) to learn how you can enroll in our Early Access program. Please review Pantheon's [Software Evaluation Licensing Terms](https://legal.pantheon.io/#contract-hkqlbwpxo) for more information about access to our software.
+
+</Alert>
+
+This section provides information on how to configure automatic platform search-replace of URLs for both subdirectory and subdomain WordPress multisites.
+
+## Requirements
+A WordPress Multisite
+
+To enable search replace between environments on a WPMS site, add 
+```yaml:title=pantheon.yml
+search_replace: true
+```
+to [pantheon.yml](/pantheon-yml).
+
+Note, if pantheon.yml is different between environments, the search_replace value in the source environment’s pantheon.yml will determine if the job will be run or not.
+
+### Subdomain WPMS
+For subdomain multisites, environments to be replaced are defined and paired in the `sites.yml` file. Currently, the file must be committed at `private/sites.yml` in the site’s git repository.
+
+See [this gist](https://gist.github.com/scottbuscemi/b051ad6510ef8494aff80d0f43afeeb2) for an example of how sites.yml might look.
+
+Search-replace will run for each domain listed in the source environment that has a matching key in the target environment.
```

Comment:
@jazzsequence, copying your comment here:
> in that case I would make sure there’s adequate info in a gh repo somewhere that the docs can link out to because there’s some nuance about how it works that should be explained more fully imo

There won't be a github repo for now, what nuance would you like clarified here?

Reply:
Discussion [here](https://pantheon.slack.com/archives/C0397K11QEQ/p1680628327865289).

tl:dr; Broadly I'd like a description of the `sites.yml` file linked as well as information and troubleshooting steps about what validation errors might trigger and how to solve them linked in here somewhere, in addition to an example valid and annotated `sites.yml` file.

Related https://github.com/pantheon-systems/pyml-validator/issues/13

Reply:
Will address as part of CMSP-319 and the docs updates to come out of that.